Dockerfile.dev

```diff
@@ -1,10 +1,10 @@
-FROM node:gallium-buster AS builder
+FROM node:hydrogen-buster AS builder
 
 ARG PKG_VERSION
 
 WORKDIR /tmp
 
-RUN npm config set unsafe-perm true
+# RUN npm config set unsafe-perm true
```

Comment:
Please delete this line is the `npm` command is not needed

Reply:
Yes this command is not needed now